src/libAtomVM/bitstring.h

```diff
@@ -122,7 +122,7 @@ static inline bool bitstring_extract_integer(term src_bin, size_t offset, avm_in
     enum BitstringFlags bs_flags, union maybe_unsigned_int64 *dst)
 {
     unsigned long capacity = term_binary_size(src_bin);
-    if (8 * capacity - offset < (unsigned long) n) {
+    if ((capacity << 3) - offset < (unsigned long) n) {
```

Comment:
I'm not sure if there is a point in doing this kind of optimization with fairly recent C compilers. I would love to check if the statement is the same. In case there is no difference, `* 8` looks more readable.